README.md

```diff
@@ -361,6 +361,12 @@ Evaluating variables in module scope may throw reference error. Possible workaro
 - use the variables of interest inside a function, and inspect
 - place an `eval()` statement inside the affected module
 
+While debugging an MJS file, variables and watch expressions do work work as expected.
+
+Resource Services do not pause as expected and therefore cannot currently be debugged.
```

Comment:
Wrap "Resource Services" with a link to https://docs.marklogic.com/guide/rest-dev/extensions#id_59188 so it's clear what that refers to.